src/Blocks/components/form/assets/state-init.js

```diff
@@ -1163,3 +1165,24 @@ export function getRestUrlByType(type, value, isPartial = false, checkRef = fals
 export function getUtilsIcons(name) {
 	return utilsManifest?.icons?.[name];
 }
+
+////////////////////////////////////////////////////////////////
+// Internal helpers.
+////////////////////////////////////////////////////////////////
+
+/**
+ * Check if local storage is available.
+ *
+ * @returns {boolean}
+ */
+function isLocalStorageAvailable() {
```

Comment:
Consider replacing this with the [MDN version](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#testing_for_availability), as it covers more edge cases.

Reply:
No need as I don't really need why the storage is unavailable. I only need to know if it will work or not. So the code is the same.